src/main/java/jp/kusumotolab/kgenprog/KGenProgMain.java

```diff
@@ -281,12 +287,37 @@ private double getNormalizedFitnessValue(final Variant variant) {
         .getNormalizedValue();
   }
 
-  private void logGAStopped(final OrdinalNumber generation) {
+  private DecimalFormat createDecimalFormat() {
+    return new DecimalFormat("#.###");
+  }
+
+  private void logGAStopped(final OrdinalNumber generation, final String time,
+      final ExitStatus exitStatus) {
     final StringBuilder sb = new StringBuilder();
     sb//
-        .append("GA stopped at the era of ")
-        .append(generation.toString())
-        .append(" generation.");
+        .append("Overall")
+        .append(System.lineSeparator())
+        .append("Reached generation = ")
+        .append(generation.intValue())
+        .append(System.lineSeparator())
+        .append("Time elapsed = ")
+        .append(time)
+        .append(System.lineSeparator())
+        .append("Exit status = ")
+        .append(exitStatus.getCode());
     log.info(sb.toString());
   }
+
+  private enum ExitStatus {
+    SUCCESS("SUCCESS"), FAIL_GEN("FAIL(maximum generation)"), FAIL_TIME("FAIL(time limit)");
```

Comment:
- successの対義語はfailureかな？
- 定数名は中途半端に略すより，全部書き切った方が分かりやすいと思う
   - `FAILURE_GENERATION`あるいは`FAILURE_MAXIMUM_GENERATION`
   - `FAILURE_TIME`あるいは`FAILURE_TIME_LIMIT`
- めっちゃ細かいけど，括弧の前に空白あった方がいいかも

```suggestion
    SUCCESS("SUCCESS"), FAILURE_GENERATION("FAILURE (maximum generation)"), FAILURE_TIME("FAILURE (time limit)");
```